docs/TempUG.md

```diff
@@ -545,93 +546,118 @@ Please use show lessonIndex before deleting task!
 
 ### Edit Feature
 
-About the feature (generally that is similar across states)
-
-Format: `command COMPULSORY [optional]` (if same command format across states)
-* Format info 1
-* Format info 2
-
-<box type="tip" seamless> 
-
-**Tips:**
-- Tip 1
-- Tip 2
-
-</box>
-
+Edits the specified item in the student/schedule list of the application.
```

Comment:
Maybe should change to the ___STUDENTS list___ and ___SCHEDULE list___?

Reply:
Thank you for pointing that out. Required update made.